src/GraphQL.Conventions/Types/Resolution/ObjectReflector.cs

```diff
@@ -8,357 +8,360 @@
 
 namespace GraphQL.Conventions.Types.Resolution
 {
-    class ObjectReflector
+    partial class TypeResolver
```

Comment:
Can you leave the `ObjectReflector` class at the top level in this file, and create the partial class for `TypeResolver` in a separate one?

Reply:
If we'd leave `ObjectReflector` not nested into `TypeResolver` it would impossible to make it protected and so to use as protected field of `TypeResolver`.
So it should be either protected and nested or public (in this case partial can be reverted)

Reply:
Just keen to keep the scope of this PR as small as possible, as I've got some hardening work for the API in progress already. So no worries about it remaining `public` for now.

Thanks

Reply:
Change to `public` and reverted `partial`